guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -77,8 +77,8 @@ repo --name=centos --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$ba
 repo --name=centos-updates --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
 repo --name=epel7 --mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
 repo --name=centos-sclo-rh --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
-repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/{ProductVersion}/el7/$basearch/
-repo --name=foreman-plugins-el7 --baseurl=http://yum.theforeman.org/plugins/{ProductVersion}/el7/$basearch/
+repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/{ProjectVersion}/el7/$basearch/
```

Comment:
```suggestion
repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/releases/{ProjectVersion}/el7/$basearch/
```

This URL only works for nightly, but not for any releases.

Reply:
Good find. "fun" fact, it was already broken before.